core/new-gui/src/app/common/service/workflow-persist/workflow-persist.service.ts

```diff
@@ -119,6 +121,24 @@ export class WorkflowPersistService {
       );
   }
 
+  /**
+   * updates the description of a given workflow, the user in the session must own the workflow.
```

Comment:
I don't think there need to be this requirement. Shared users can also update the description.